.github/workflows/basesystem.yml

```diff
@@ -1,62 +1,123 @@
 name: Build basesystem
 # Builds the Storm basesystem Docker images and deploys them to Dockerhub
+# Build is distributed across multiple runners based on:
+# https://docs.docker.com/build/ci/github-actions/multi-platform/#distribute-build-across-multiple-runners
 
 on:
   # needed to trigger the workflow manually
   workflow_dispatch:
 
+env:
+  IMAGE: movesrwth/storm-basesystem
+
 jobs:
-  buildDistro:
-    name: Build for ${{ matrix.distro.name }}
+  build:
+    name: Build ${{ matrix.image.name }} on ${{ matrix.platform }}
     runs-on: ubuntu-latest
```

Comment:
what is this variable for?

Reply:
I am note sure which variable you mean.
The output on the Github website (given by `name`) will be something like "Build debian-12 on linux/arm64".
The name `build` is just used to denote the job and used to record that the job `merge` later on depends on the job ` build`.
Variable `ubuntu-latest` is the base system where the Docker runs.

Reply:
Ah, so ubuntu-latest hosts the docker container and Debian 12 is the guest system? Thanks for clarifying

Reply:
Yes, exactly.